### PR TITLE
Fixing issues on #26 

### PR DESCRIPTION
**Pull request by syedautherabbas**

### CODE REVIEW

**Comment by antoniolofiego** [comment]
The base branch target was wrong, fixed it to staging.

File: test_API/test_get_user_by_Id/test_get_user_by_ID.py [hunk 1]
```diff
@@ -52,5 +50,6 @@ def dynamodb_table(dynamodb):
 def test_get_user_by_ID(dynamodb,dynamodb_table):
     table=dynamodb_table
     table.put_item(Item={"github_username":'johndoe'})
-    response = table.get_item(Key={'github_username': 'johndoe'})
-    assert len(response['Item']) > 0    
\ No newline at end of file
+    #response = table.get_item(Key={'github_username': 'johndoe'}) local test
+    response=get_user_by_Id.get_user_details_by_Id(table,'johndoe')
+    assert True if 'github_username' in response else False
```
Comment (antoniolofiego) [inline test_API/test_get_user_by_Id/test_get_user_by_ID.py]: This could be changed to `assertIn('github_username', response')`

**Review by antoniolofiego — COMMENTED** [review]
This looks good so far, much cleaner handling of the DDB calls! Have not had a chance to test locally yet, but I will do so as soon as possible. I will wait to merge until we have the CI pipeline in place to automate the tests and this passes.

**Comment by what-name** [comment]
Looks great! Please put `__pycache__` into `.gitignore` and it can get merged.

**Comment by antoniolofiego** [comment]
> Looks great! Please put `__pycache__` into `.gitignore` and it can get merged.

Please, look at #27 before merging anything. We need to set up automated testing before merging new things.

**Comment by what-name** [comment]
Close due to migration of the whole staging branch to [100DaysOfCloudBackend](https://github.com/100DaysOfCloud/100DaysOfCloudBackend).

@syedautherabbas please remove the `__pycache__` folder and resubmit this PR to the new repo after it has been populated.

**Pull request closed without merging**